<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagHandler</class>
 <widget class="QWidget" name="TagHandler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="optLayout"/>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>tags</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="selTagsLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
